bindings/pyroot_experimental/PyROOT/src/TTreePyz.cxx

```diff
@@ -236,3 +236,155 @@ PyObject *PyROOT::SetBranchAddressPyz(PyObject * /* self */, PyObject *args)
    // Not the overload we wanted to pythonize, return None
    Py_RETURN_NONE;
 }
+
+// Try ( const char*, void*, const char*, Int_t = 32000 )
+PyObject *TryBranchLeafListOverload(int argc, PyObject *args)
+{
+   PyObject *treeObj = nullptr;
+   PyObject *name = nullptr, *address = nullptr, *leaflist = nullptr, *bufsize = nullptr;
```

Comment:
why 2 lines?

Reply:
The first line is the tree object on which `Branch` is invoked, the second line contains the possible parameters of `Branch` in the order they appear in the signature. Seemed cleaner to me than just all in one line.